glean-core/pings.yaml

```diff
@@ -83,6 +83,15 @@ events:
     - https://bugzilla.mozilla.org/show_bug.cgi?id=1512938#c3
   notification_emails:
     - glean-team@mozilla.com
+  reasons:
+    startup: |
+      The ping was submitted at startup. The events ping is always sent at
```

Comment:
This should mention that it is submitted on startup if it contains pending events.